AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -119,30 +144,32 @@ + (NSUInteger)parallelProcessorCount
 
 #pragma mark - Cell Layout
 
-- (void)batchLayoutNodesFromContexts:(NSArray<ASIndexedNodeContext *> *)contexts batchCompletion:(ASDataControllerCompletionBlock)batchCompletionHandler
+- (void)batchLayoutNodesFromContexts:(NSArray<ASIndexedNodeContext *> *)contexts batchSize:(NSUInteger)batchSize batchCompletion:(ASDataControllerCompletionBlock)batchCompletionHandler
```

Comment:
Nit: When adding new integer variables, let's standardize on `NSInteger` because it's simpler and the Swift import is way easier.